corehq/apps/userreports/README.md

```diff
@@ -401,6 +413,121 @@ The `date_expression` can be any valid expression, or simply constant
 }
 ```
 
+
+#### Filter, Sort, Map and Reduce Expressions
+
+We have follwoing expressions that act on a list of objects or list of lists. The list to operate on is specified by `items_expression`. This can be any valid expression that returns a list. If the `items_expression` doesn't return a valid list, these might either fail or return one of empty list or `None` value.
+
+##### map_items Expression
```

Comment:
is there a reason we went with `map_items` as opposed to just `map`? don't think it matters too much, just wondering